Index: src/main/java/ASESpaghettiCode/UserServer/Model/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ASESpaghettiCode/UserServer/Model/User.java b/src/main/java/ASESpaghettiCode/UserServer/Model/User.java
new file mode 100644
--- /dev/null	(date 1679093959306)
+++ b/src/main/java/ASESpaghettiCode/UserServer/Model/User.java	(date 1679093959306)
@@ -0,0 +1,2 @@
+package ASESpaghettiCode.UserServer.Model;public class user {
+}
Index: src/main/java/ASESpaghettiCode/UserServer/Controller/UserController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ASESpaghettiCode/UserServer/Controller/UserController.java b/src/main/java/ASESpaghettiCode/UserServer/Controller/UserController.java
new file mode 100644
--- /dev/null	(date 1679093959304)
+++ b/src/main/java/ASESpaghettiCode/UserServer/Controller/UserController.java	(date 1679093959304)
@@ -0,0 +1,14 @@
+package ASESpaghettiCode.UserServer.Controller;
+
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user")
+public class userController {
+    @RequestMapping("/{userId}")
+    public String getUser(@PathVariable("userId") String userId) {
+        return "User server is working."+ "This is "+userId;
+    }
+}
